Firmware/Marlin_main.cpp

```diff
@@ -9877,7 +9877,11 @@ void long_pause() //long pause print
 
     // Stop heaters
     heating_status = HeatingStatus::NO_HEATING;
+#ifdef HEATER_LOWER_TEMP
+    setTargetHotend(target_temperature[active_extruder]- HEATER_LOWER_TEMP);
```

Comment:
I think we need some protection here to cover edges cases. This can produce negative values: `HEATER_LOWER_TEMP > target_temperature[active_extruder]`

What happens if we are coming from a temperature error? `ThermalStop()` can sometimes pause prints.

Reply:
Very good point, thanks. Fan check, thermal check and ThermalStop call `lcd_pause_print` --> `LcdCommands::LongPause` -> long_pause`

Reply:
Might want to subtract only if the target temperature is >= HEATER_LOWER_TEMP. Otherwise you might underflow the target temperature in even more scenarios where the target is a small number